perf_test/sparse/KokkosSparse_spmv_struct_tuning.cpp

```diff
@@ -92,6 +92,48 @@ void print_help() {
   printf("  --print-lp         : Print launch parameters to screen.\n");
 }
 
+template<typename graph_type>
+struct copy_crs_data {
+  using execution_space = typename graph_type::device_type::execution_space;
+  using cusparse_int_type = typename Kokkos::View<int*,
+						  typename graph_type::entries_type::array_layout,
+						  typename graph_type::device_type>;
+
+  // Dispatch tags
+  struct rowPtrTag{};
+  struct colIndTag{};
+
+  typename graph_type::row_map_type::const_type Arowptr;
+  typename graph_type::entries_type::const_type Acolind;
+  cusparse_int_type cusparse_Arowptr, cusparse_Acolind;
+
+  copy_crs_data(typename graph_type::row_map_type::const_type Arowptr_,
+		typename graph_type::entries_type::const_type Acolind_,
+		cusparse_int_type cusparse_Arowptr_,
+		cusparse_int_type cusparse_Acolind_) :
+    Arowptr(Arowptr_), Acolind(Acolind_),
+    cusparse_Arowptr(cusparse_Arowptr_),
+    cusparse_Acolind(cusparse_Acolind_) {};
+
+  void doCopy() {
+    Kokkos::RangePolicy<execution_space, rowPtrTag> rowPtrPolicy(0, Arowptr.extent(0));
```

Comment:
See `Tpetra::Details::copyOffsets`.  That would be a useful kokkos-kernels utility.